tests/api2/test_zzzz_stig.py

```diff
@@ -139,18 +151,30 @@ def setup_stig(two_factor_full_admin_as_builtin_admin):
                     yield {
                         'connection': c,
                         'user_obj': user_obj,
-                        'secret': secret
+                        'secret': secret,
+                        'aal': aal
                     }
                 finally:
-                    c.call('system.security.update', {'enable_fips': False, 'enable_gpos_stig': False}, job=True)
+                    # Restore default security, user and network settings
+                    c.call('system.security.update', {
+                        'enable_fips': False, 'enable_gpos_stig': False,
+                        'min_password_age': None, 'max_password_age': None,
+                        'password_complexity_ruleset': None, 'min_password_length': None,
+                        'password_history_length': None
+                    }, job=True)
+
```

Comment:
Nothing wrong with this, but another option would have been to **read** the config before the `do_stig_auth` call. Meh.